templates/compositions/ec-component-language-list/language-list-selector.html.twig

```diff
@@ -39,6 +46,12 @@
 
 {# Internal logic - Process properties #}
 
+{% if eu_languages is not empty and eu_languages is iterable %}
+  {% set items = eu_languages %}
+  {% set eu_category = 'EU official languages'|t %}
```

Comment:
Do we have translations for these strings we can ship? There was that huge excel file with translations somewhere.

Reply:
No translation for this combination of strings in that xls.